packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -850,6 +856,70 @@ const BigQueryDatasourceSpecificationEditor = observer(
   },
 );
 
+const SpannerDatasourceSpecificationEditor = observer(
+  (props: {
+    sourceSpec: SpannerDatasourceSpecification;
+    isReadOnly: boolean;
+  }) => {
+    const { sourceSpec, isReadOnly } = props;
+    return (
+      <>
+        <ConnectionEditor_StringEditor
+          isReadOnly={isReadOnly}
+          value={sourceSpec.projectId}
```

Comment:
Let's be consistent and use `"..."` instead of `{'...'}` for these. Thanks!

Reply:
I guess thats for a line below

Reply:
for all of them actually, I prefer us using `""` form to `{'...'}`

Reply:
Changed here and downward